src/parse/cloud/main.js

```diff
@@ -49,5 +49,8 @@ module.exports = (Parse, event) => {
       }
       logger.info(`Parse correctly init`);
     })
-    .catch((err) => logger.info(`Issue to init Parse : ${err}`));
+    .catch((err) => {
+      logger.error(`Issue to init Parse : ${err}`);
+      throw err;
```

Comment:
the recommended way of throwing an error (eslint) is `throw new Error(err)`

Reply:
Are you refering to the [no-throw-literal](https://eslint.org/docs/rules/no-throw-literal) rule? This rule is not enabled in the current project, but in this case `err` is already an error object (i throw a string in the code i use to validate the json in src/parse/cloud/setBeforeSave.js).